core/src/main/java/jenkins/DailyCheck.java

```diff
@@ -0,0 +1,68 @@
+package jenkins;
+
+import hudson.Extension;
+import hudson.Main;
+import hudson.model.AsyncPeriodicWork;
+import hudson.model.DownloadService;
+import hudson.model.TaskListener;
+import hudson.model.UpdateSite;
+import hudson.util.FormValidation;
+import jenkins.model.Jenkins;
+import org.jenkinsci.Symbol;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.kohsuke.stapler.HttpResponse;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Extension
+@Restricted(NoExternalUse.class)
+@Symbol("updateCenterCheck")
+public final class DailyCheck extends AsyncPeriodicWork {
```

Comment:
For anyone else wondering why this has changed, it used to be in `DownloadSettings`
https://github.com/jenkinsci/jenkins/pull/3970/files#diff-7148b4537324284098f8e7da7de98851L55

Reply:
Addressed in https://github.com/jenkinsci/jenkins/pull/3970/commits/2bbc99c6a395c102d4387d8ea16c74ad7b0faa4a#diff-bf2e22703bf0c4135b90223a84a3ccaeR46